commands/describe.go

```diff
@@ -109,9 +109,16 @@ func runDescribe(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func getFunctionURLs(gateway string, functionName string) (string, string) {
+func getFunctionURLs(gateway string, functionName string, functionNamespace string) (string, string) {
 	gateway = strings.TrimRight(gateway, "/")
-	return gateway + "/function/" + functionName, gateway + "/async-function/" + functionName
+	url := gateway + "/function/" + functionName + "." + functionNamespace
+	asyncURL := gateway + "/async-function/" + functionName + "." + functionNamespace
+
+	// Remove dot for functions without namespaces.
+	url = strings.TrimRight(url, ".")
```

Comment:
Might be better to add conditionally, rather than mutating the string and then trimming, this seems like a bit of a hack.

Would something like this work? The existing tests should continue to work. 

```
if len(functionNamespace) > 0 {

}
```

Reply:
Yes, I think it would work. Having an if statement also makes the comment "// Remove dot..." unnecessary. I'll remove that comment.

Reply:
I ended up writing a simpler `if functionNamespace == "" {...}`, which doesn't have a `len()` function call.